qprofiler2/src/org/qcmg/qprofiler2/util/XmlUtils.java

```diff
@@ -20,47 +20,49 @@
 import htsjdk.samtools.SAMSequenceRecord;
 
 public class XmlUtils {
-	public final static String OTHER = "others";
-	
-	public static final String metrics = "Metrics";
-	public static final String readGroupsEle ="readGroups";
-	public static final String variableGroupEle = "variableGroup";
-	public static final String sValue = "value";
-	public static final String sBin = "closedBin";
-	public static final String metricsEle = "sequence" + metrics;
-	public static final String sName = "name";
-	public static final String sCount = "count";
-	public static final String sPercent = "percent";
-	public static final String sTally = "tally";
-	public static final String sTallyCount = "tallyCount";
-	public static final String sStart = "start";
-	public static final String sEnd = "end";
-	public static final String Scycle = "cycle";
-	public static final String baseCycleEle = "baseCycle";
-	public static final String recordEle ="record";
-	public static final String discardReads = "discardedReads";
-		
-	public static final String seqBase = "seqBase";
-	public static final String seqLength = "seqLength";
-	public static final String badBase = "badBase";
-	public static final String qualBase = "qualBase";
-	public static final String qualLength = "qualLength";	
-	public static final String firstOfPair = "firstReadInPair"; 
-	public static final String secondOfPair = "secondReadInPair";
+	public static final String OTHER = "others";
 	public static final String UNKNOWN_READGROUP = "unkown_readgroup_id";	
-	public static final String summary = "bamSummary";
 	
+	public static final String METRICS = "Metrics";
+	public static final String READGROUP_ELE ="readGroups";
+	public static final String VARIABLE_GROUP_ELE = "variableGroup";
+	public static final String VALUE = "value";
+	public static final String COLSED_BIN = "closedBin";
```

Comment:
typo

Reply:
Sorry Christina, I should have been more specific - it was `COLSED_BIN` that I was hoping that you would change